### DISCUSSION TITLE
Any files is updated when a new client is added to Jack?

### DISCUSSION THREAD

**Discussion by fernando-inf** [jackaudio/jack2#802] [Q&A]
`watch -x -n 1 find /JACKFOLDER?/*.* -mmin -1 -exec /home/jack_connect.sh \; &`

Hi, I am trying to combine in DEBIAN 11 the `watch` and `find` command to automatically execute a connection when a new client is added to jack.

For this I need to know if when adding a new client there is a folder to look for so that `find` will look for changes in it and execute the connection. Thanks.

**Comment by falkTX** [comment]
Dont do this with files, just use tools such as `jack_evmon`.

Also, this is not really a bug/ticket, so I am converting it to a discussion topic.

**Comment by fernando-inf** [comment]
That's fine thanks. But I'm a programming beginner, I don't know exactly how I should use `jack_evmon`. Maybe create a custom Jack App based on evmon ?.

It would also be easier for me if I do it with `watch` and `find`. I see that `jack_evmon` uses 106MB RAM, while `watch` uses 2.5MB, I need to minimize RAM usage.

**Reply by falkTX** [reply]
just run the tool with the needed arguments, it is all you need. `man jack_evmon` is your staring point.
and whatever you are using to monitor RAM is wrong, or perhaps you are just reading it incorrectly. This is a stupid simple basic CLI tool, impossible for it to use more than 1Mb of RAM

**Reply by falkTX** [reply]
wait jack_evmon has no args.. I was quite sure it did, sorry

**Comment by fernando-inf** [comment]
That's right, it doesn't show arguments, it only shows messages when there is a change in jack.

![evmon](https://user-images.githubusercontent.com/85811973/138767538-0b34cedf-bce5-4e48-b876-e9df10fc130e.png)

**Comment by lucmp** [comment]
Interesting topic.

```
$ man jack_evmon 
No manual entry for jack_evmon
See 'man 7 undocumented' for help when manual pages are not available.
```

So how does jack_evmon work?
